packages/mongodb-memory-server-core/src/util/__tests__/MongoBinaryDownloadUrl.test.ts

```diff
@@ -1723,6 +1723,24 @@ describe('MongoBinaryDownloadUrl', () => {
       expect(du.getLegacyVersionString).toHaveBeenCalledTimes(1);
       expect(ret).toBe('');
     });
+
+    it('should give a warning about "arch"', () => {
```

Comment:
that is actually the wrong place, this `describe` block tests the that `getLinuxOSVersionString` behaves correctly, distro specific tests are in `MongoBinaryDownloadUrl -> getDownloadUrl() -> for linux` and there already exist similar tests for arch called `for arch` and `for unpopular arch`, though i would probably later move them into their own describe block